datahub-web-react/cypress/integration/Search.spec.ts

```diff
@@ -1,36 +0,0 @@
-import { createLoginUsers } from '../../src/graphql-mock/fixtures/user';
-import { makeServer } from '../../src/graphql-mock/server';
-import { login, logout } from '../helper/authHelper';
-
-describe('Search', () => {
-    let server;
-
-    beforeEach(() => {
-        server = makeServer('test');
-        createLoginUsers(server);
-    });
-
-    afterEach(() => {
-        server.shutdown();
-    });
-
-    describe('given the home page is loaded', () => {
-        describe('when the user enters a keyword in the search field and results found and the first item is selected from the search result dropdown', () => {
-            it('then the search result page should be displayed with the Task tab be selected and the selected item be displayed', () => {
```

Comment:
same question here (or at least equivalent tests)

Reply:
we have test for search